csrc/scheduler/transpose.cpp

```diff
@@ -62,7 +62,7 @@ class DomainMap : public pointwise_utils::DomainMap {
     const auto& root_dom = tv->getRootDomain();
     IterDomain* mapped_id = nullptr;
     for (auto i : c10::irange(root_dom.size())) {
-      if (ca_map_.idGraph().permissiveNodes().permissiveAreMapped(
+      if (ca_map_.idGraph().permissiveResizeNodes().permissiveAreMapped(
```

Comment:
@naoyam We should use the most permissive mapping here. I think this is a bug unrelated to this change but being exposed for some reason. I don't know in detail why this was not causing any trouble before.

Reply:
That seems to be the case. I'll check why it wasn't failing.